app/jobs/resolution_proofing_job.rb

```diff
@@ -190,6 +225,9 @@ def resolution_proofer
     @resolution_proofer ||=
       if IdentityConfig.store.proofer_mock_fallback
         Proofing::Mock::ResolutionMockClient.new
+      # We run ddp ThreatMetrix in serial before resolution proofer. In future we can replace here
+      # elsif IdentityConfig.store.replace_lexisnexis_rdp_instant_verify_with_ddp
+      #   lexisnexis_ddp_proofer
```

Comment:
This is a ways away, let's not leave this commented out code if we don't need it?

Reply:
ok